lib/generator/command_line.rb

```diff
@@ -9,33 +7,23 @@ def initialize(paths)
     end
 
     def parse(args)
-      parser = GeneratorOptparser.new(args, @paths)
+      parser = GeneratorOptparser.new(args, paths)
       @options = parser.options
-      generator if parser.options_valid?
+      generate if parser.options_valid?
```

Comment:
So it is 😄  

The change is an artifact of me tugging at the code repeatedly until I arrived at what you see. Looking at it now, I don't like either. `generator if parser.options_valid?` is lousy english. `generate if parser.options_valid?` is worse, as it is misleading. Maybe `#build_generator`?